Assets/Scripts/Models/Events/HeadlineGenerator.cs

```diff
@@ -40,7 +33,7 @@ public void AddHeadline(string headline, bool displayImmediately = true, bool ke
     {
         if (keepInQueue)
         {
-            headlines.Add(headline);
+            PrototypeManager.Headline.Add(new Headline(headline));
```

Comment:
Since the headline added here will run more than 1 time it made sense to add it to the prototype. But if it should just run 1 time, then HeadlineGenerator should have a Queue of Headlines and grab from them.